openpype/hosts/fusion/api/menu.py

```diff
@@ -59,6 +67,17 @@ def __init__(self, *args, **kwargs):
         publish_btn = QtWidgets.QPushButton("Publish...", self)
         manager_btn = QtWidgets.QPushButton("Manage...", self)
         libload_btn = QtWidgets.QPushButton("Library...", self)
+
+        build_workfile_from_template_btn = QtWidgets.QPushButton(
+            "Build workfile from template...", self
+        )
+        create_placeholder_btn = QtWidgets.QPushButton(
+            "Creat placeholder...", self
+        )
+        update_placeholder_btn = QtWidgets.QPushButton(
+            "Update placeholder...", self
+        )
```

Comment:
Should we move this into a submenu or alike?

This is what it looks like in Maya:
![afbeelding](https://user-images.githubusercontent.com/2439881/228672352-2e518101-8a9a-4503-8728-291ba0c06f89.png)

Also like in Maya, let's move them to the bottom below reset framerange, etc.

Reply:
Yep, that's the way to go but my first focus is to get everything up and running and after clean up the code and the menu UI. After that I'll change from draft to review :)